src/pages/home/report/ReportActionCompose/Suggestions.js

```diff
@@ -25,10 +28,24 @@ const defaultProps = {
  *
  * @returns {React.Component}
  */
-function Suggestions({isComposerFullSize, value, setValue, selection, setSelection, updateComment, composerHeight, forwardedRef, resetKeyboardInput, measureParentContainer}) {
+function Suggestions({
+    isComposerFullSize,
+    value,
+    setValue,
+    selection,
+    setSelection,
+    updateComment,
+    composerHeight,
+    forwardedRef,
+    resetKeyboardInput,
+    measureParentContainer,
+    isAutoSuggestionPickerLarge,
+}) {
     const suggestionEmojiRef = useRef(null);
     const suggestionMentionRef = useRef(null);
 
+    const getSuggestions = useCallback(() => suggestionEmojiRef.current.getSuggestions() || suggestionMentionRef.current.getSuggestions(), []);
```

Comment:
This caused a regression as null handling is missing:

https://github.com/Expensify/App/issues/31027#issuecomment-1801259985